lua/gatekeeper_config.lua

```diff
@@ -10,7 +10,7 @@ function gatekeeper_init()
 	-- When gatekeeper_server is true,
 	-- Gatekeeper will run as a Gatekeeper server.
 	-- Otherwise, it will run as a grantor server.
-	local gatekeeper_server = true
+	local gatekeeper_server = false
```

Comment:
To avoid the pull requests changing this variable back and forth, and since it is the only change in this file, could you drop this change?